src/VecSim/spaces/spaces.h

```diff
@@ -31,10 +31,4 @@ CalculationGuideline FP64_GetCalculationGuideline(size_t dim);
 void SetDistFunc(VecSimMetric metric, size_t dim, dist_func_t<float> *index_dist_func);
 void SetDistFunc(VecSimMetric metric, size_t dim, dist_func_t<double> *index_dist_func);
 
-// Useful numbers
-const unsigned int LOG_BASE2_16 = 4;
```

Comment:
Why?

Reply:
well i added it in the first dist_func PR but I didnt use it and forgot to remove it